plugins/slurm/internal/defaults/dump_adapters.go

```diff
@@ -0,0 +1,28 @@
+package defaults
+
+import (
+	"context"
+
+	"buf.build/gen/go/cedana/cedana/protocolbuffers/go/daemon"
+	criu_proto "buf.build/gen/go/cedana/criu/protocolbuffers/go/criu"
+	"github.com/cedana/cedana/gen/plugins/slurm"
```

Comment:
Where does this come from? Building locally, this cannot be found.

Can you simply do a `go get` to fetch the newly generated module from buf.build for you `cedana-api` branch? 

Link: https://buf.build/cedana/cedana/sdks/feature/ced-1524-commit-a-slurm-interface-for-the-cedana-propagator:protocolbuffers/go?version=v1.36.6

Let me know if you can’t access it.